modules/decrypt-node/test/decrypt.test.ts

```diff
@@ -83,4 +83,21 @@ describe('decrypt', () => {
       { encoding: 'base64' }
     )).to.rejectedWith(Error, 'Invalid Signature')
   })
+
```

Comment:
No unit tests for ParseHeaderStream?

Reply:
Specifically, no.  But implicitly, the `stream` test in this file will exercise just about everything.

Reply:
Unless there is some reason why unit tests for ParseHeaderStream specifically would not be useful, we should add them.
Either in this PR, or (because the logic added in this PR seems to be sufficiently tested below) tracked in a new issue.

Reply:
#188